configs/cascade_rcnn/metafile.yml

```diff
@@ -12,8 +12,13 @@ Collections:
         - RPN
         - ResNet
         - RoIAlign
-    Paper: http://dx.doi.org/10.1109/tpami.2019.2956516
+    Paper:
+      URL: http://dx.doi.org/10.1109/tpami.2019.2956516
+      Title: 'Cascade R-CNN: Delving into High Quality Object Detection'
     README: configs/cascade_rcnn/README.md
+    Code:
+      URL: https://github.com/open-mmlab/mmdetection/blob/v0.5.7/mmdet/models/detectors/cascade_rcnn.py#L15
```

Comment:
The version and code should be the version that trains this model rather than the time it released